src/SpectatorView.Unity/Assets/SpectatorView/Scripts/Compositor/TextureManager.cs

```diff
@@ -239,7 +248,7 @@ private void SetupCameraAndRenderTextures()
         private void OnPreRender()
         {
             Graphics.Blit(CurrentColorTexture, colorRGBTexture, CurrentColorMaterial);
-            Graphics.Blit(colorRGBTexture, spectatorViewCamera.targetTexture);
+            Graphics.Blit(null, spectatorViewCamera.targetTexture, textureClearMat);
         }

```

Comment:
nit: Blit'sing a null texture isn't super intuitive. Could you add a comment calling out why the source is null?